### ISSUE TITLE
Style guide | Text accessibility 

### ISSUE THREAD

**Issue by gnakm** [department-of-veterans-affairs/caseflow#4414]
## Description

Right now text accessibility section is nested under Color when it should be under Typography

## AC
- [ ] Change so that the text accessibility on the side nav is found under typography and also nested the same way on page.
